src/backend/distributed/utils/citus_ruleutils.c

```diff
@@ -774,6 +739,79 @@ pg_get_table_grants(Oid relationId)
 }
 
 
+/*
+ * generate_qualified_relation_name
+ *		Compute the name to display for a relation specified by OID
```

Comment:
Correct, the function was in ruleutils_95.c and I forgot to update the comments after moving it to citus_ruleutils.